src/integ_test_resources/android/sdk/integration/cdk/README.md

```diff
@@ -20,25 +20,39 @@ npm install -g aws-cdk
 Ensure that you have [credentials in your environment sufficient to run
 the CDK](https://docs.aws.amazon.com/cdk/latest/guide/getting_started.html#getting_started_credentials).
 
-Install CDK dependencies:
+Install CDK dependencies from the root directory of the repo:
 ```console
 # Best to do this in a virtualenv
-pip install -r requirements.txt
+pip3 install -r requirements.txt
 ```
 
-Generate a Pinpoint stack template at
+**Export credentials and region:**
+
+```console
+isengard assume <account name>
+isengard credentials <account name>
+```
```

Comment:
Remove. Don't reference internal tools in public contexts.

For this, you'll need to provide a general statement like "export appropriate role credentials to your local shell" or something.

Reply:
+1 from Jimbo

Reply:
Done